batch/test/test_dag.py

```diff
@@ -162,6 +162,7 @@ async def callback(request):
         callback_body.pop('duration')
         callback_body.pop('duration_ms')
         callback_body.pop('cost_breakdown')
+        callback_body['attributes'].pop('client_job')
```

Comment:
there's also `'name'`

Reply:
@danking name is controlled for and tested down below though